Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -51,23 +53,46 @@
+ BTWK_story_event.0114.desc_alliance_else:0 ""
+ BTWK_story_event.0114.desc_arthur:0 ""
+ BTWK_story_event.0114.desc_norwenna_dead:0 ""
+
  BTWK_story_event.0114.a:0 "To us!"  # needs reloc to something else
  BTWK_story_event.0114.b:0 "It looks like we will have to forge our own path."
  BTWK_story_event.0114.c:0 "I pray we have done enough to stave off the invasions." 
- BTWK_story_event.0114.d:0 "May these alliances prove fruitful, for all our sakes."
+ BTWK_story_event.0114.d:0 "May these new bonds prove fruitful, for all our sakes."
+##############################
 
  BTWK_story_event.0115.t:0 "A Summit in Glevum"
- BTWK_story_event.0115.desc_header:0 "The High King of Britannia, Uther Pendragon, has recently hosted a High Council in the villa of Glevum."
- BTWK_story_event.0115.desc_alliance:0 "Word has reached you that through arranging a marriage between Norwenna - the widow of the late Prince of Dumnonia, Mordred - and (SPOUSE), Uther has secured an alliance between his realm, and (NEW REALM). A number of guardians have also been named for the Edling Mordred. Chiefly among them is Tewdric, King of Gwent, the legendary druid Merlin, and Uther's own son, the renowned warlord Prince Arthur. Such a web of alliances will surely fortify Dumnonia against its enemies until the young Prince reaches maturity."
- BTWK_story_event.0115.desc_arthur:0 "Word has reached you that, despite the misgivings he holds for his bastard son, a marriage has been arranged between Prince Arthur, and his late brother's widow, Norwenna. With Arthur's arrangment as the Edling Prince's foster father, his return to the shores of Britannia from the Armorican frontlines is imminent. A number of guardians have also been named for the young Mordred. Chiefly among them are Tewdric, King of Gwent, and the legendary druid Merlin. While Uther was unsuccessful in uniting the British warlords towards their continued survival, the impending return of Arthur may be enough to give Dumnonia some pause from the assaults of its enemies - for a time at least."
- BTWK_story_event.0115.desc_norwenna_dead:0 "With Norwenna not long passing after her husband - the late Prince Mordred - Uther's intention of an alliance forged through marriage was not to be, leaving Dumnonia more exposed than he would've liked. However, some loyalists to the High King and his young heir were still to be found for its defence. Chiefly among them is Tewdric, King of Gwent, the legendary druid Merlin, and Uther's own son, the renowned warlord Prince Arthur. While Uther was unsuccessful in uniting the British warlords towards their continued survival, this little web of alliances will at least give Dumnonia some reprieve from the assaults of its enemies - for a time at least."
+ BTWK_story_event.0115.desc_header:0 "The High King of Britannia and King of [GetTitleByKey('k_dumnonia').GetNameNoTier], Uther Pendragon, has recently hosted a High Council in the old Roman town of Glevum."
```

Comment:
- "old Roman town of Glevum" -> "great hall of Glevum", old Roman town implies that the Romans have been gone for much longer than they would have at this point